zephyr/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Nordic WiFi driver for nRF52840 and nRF5340
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config WPA_SUPP
+    bool "WPA WPA_SUPP"
+    select EARLY_CONSOLE
```

Comment:
Why does supplicant select this config, is it really essential for supplicant operation?

Reply:
dropped